cli/command/environment.go

```diff
@@ -78,17 +73,13 @@ func (e *EnvironmentCommand) Command() cli.Command {
 			},
 			{
 				Name:      "set-scale",
```

Comment:
[nit] would you consider using `scale` instead of `set-scale`? It's a personal preference, but I like that `scale` is consistent with the other single-verb commands, `list`, `delete`, etc.

Reply:
✅